CMakeLists.txt

```diff
@@ -1,39 +1,76 @@
 cmake_minimum_required(VERSION 2.8)
 
-set(PROJ_CJSON cJSON)
+set(CJSON_VERSION_MAJOR 1)
+set(CJSON_VERSION_MINOR 0)
+set(CJSON_VERSION_PATCH 0)
+set(CJSON_VERSION_SO 1)
+set(CJSON_UTILS_VERSION_SO 1)
+set(CJSON_VERSION "${CJSON_VERSION_MAJOR}.${CJSON_VERSION_MINOR}.${CJSON_VERSION_PATCH}")
+
+add_compile_options(-ansi -pedantic -Wall -Wextra -Werror -Wstrict-prototypes -Wwrite-strings)
+
+#variables for pkg-config
+set(prefix ${CMAKE_INSTALL_PREFIX})
+set(libdir "${CMAKE_INSTALL_PREFIX}/lib")
+set(version ${CJSON_VERSION})
 
+option(BUILD_SHARED_LIBS "Build shared libraries" ON)
+
+#cJSON
+set(PROJ_CJSON cJSON)
+set(CJSON_LIB cjson)
 project(${PROJ_CJSON} C)
 
 file(GLOB HEADERS cJSON.h)
 set(SOURCES cJSON.c)
 
-add_library(${PROJ_CJSON} ${HEADERS} ${SOURCES})
+add_library(${CJSON_LIB} ${HEADERS} ${SOURCES})
 if (NOT WIN32)
-	target_link_libraries(${PROJ_CJSON} m)
+    target_link_libraries(${CJSON_LIB} m)
 endif()
 
-set(PROJ_CJSON_UTILS cJSON_utils)
+configure_file("${cJSON_SOURCE_DIR}/libcjson.pc.in"
+    "${cJSON_BINARY_DIR}/libcjson.pc" @ONLY)
 
+install(TARGETS ${CJSON_LIB} DESTINATION lib${LIB_SUFFIX})
```

Comment:
I'm not sure actually. This was added during the time that Kevin Branigan was still maintaining the git repository. I will look into the module GNUInstallDirs.

Thanks for pointing this out.